addons/mail/models/mail_mail.py

```diff
@@ -541,7 +541,10 @@ def send(self, auto_commit=False, raise_exception=False):
                     len(batch_ids), mail_server_id)
             finally:
                 if smtp_session:
-                    smtp_session.quit()
+                    try:
+                        smtp_session.quit()
+                    except Exception:
```

Comment:
Shouldn't we be a bit more precise, like catching real errors you spotted on sentry ?

Reply:
More precise error is `SMTPServerDisconnected`.

There is a similar discussion about whether we should use broad exception or specific one.
Let's finish the discussion there first https://github.com/odoo/odoo/pull/123644#issuecomment-1665324921

Reply:
I am more in favor of being specific, allowing to really know what are the issues people faced... and maybe someday find real issues to handle differently.